meetings/working-groups/extensions/Extension-API-docs.md

```diff
@@ -0,0 +1,150 @@
+
+## Docs presentation for C# 14 extension members
+
+The presentation for C# 14 extensions needs to account for several new types of extension members:
+
+- Extension properties.
+- Extension indexers.
+- Extension operators.
+- Extension members whose receiver is a *type* rather than an *instance of a type*. In other words, extension members that are meant to accessed as though they are static members of the extended type. These can include methods, properties, and indexers.
+
+The current format for extension methods should be extended as follows:
+
+### Extension member prototypes
+
+When an extension member prototype is shown, the format should show the extension container:
+
+```csharp
+extension<T>(IEnumerable<T> source)
+{
+    public bool IsEmpty { get; }
+}
+```
+
+The `source` parameter is referred to as the *receiver parameter*, or *receiver*. The *receiver parameter* may be an instance, as shown above, or it may be a type, as in the following:
+
+```csharp
+extension<T>(IEnumerable<T>)
+{
+    public static IEnumerable<T> Create(int size, Func<int, T> factory);
+}
+```
+
+This presentation enables the following:
+
+- Readers see the new extension syntax when consuming new extensions, driving awareness and adoption.
+- Readers can quickly distinguish a new-style extension (noted by the receiver parameter), and existing extension methods (noted by the `this` modifier on the first parameter).
+- The `extension` container indicates that the member is an extension member.
+- The parameter on the `extension` node indicates the type extended, and provides a key to know if the member is intended to extend an instance or extend the type.
+- The prototype only includes the `static` modifier if it would be present in source, as in an operator declaration:
+  ```csharp
+  extension<T>(IEnumerable<T> source)
+  {
+      public static IEnumerable<T> operator + (IEnumerable<T> left, IEnumerable<T> right);
+  }
+  ```
+
+### Extension members in the extended type's page
+
+The API docs build system generates the section on the type page for the extended type that lists all extension members. This section should have sub-sections for *extension methods*, *extension properties*, and *extension operators*. Extension indexers should follow the format for indexers, and be listed as an `Item[]` property. There isn't a `this` modifier on the first parameter. In fact, the receiver is declared on the extension, not the member. The prototypes in this section should expand to show the `extension` container, as follows:
+
+- The prototypes are displayed as described in the previous section.
+- Otherwise, the format is consistent with the current format:
+  - Each overload has a separate entry in the section.
+  - The signatures indicate if the method is a specialization (`Sum(this IEnumerable<int>)`) or generic (`Where<T>(this IEnumerable<T> source, Func<T, bool> predicate)`).
```

Comment:
The signatures here are of classic extension methods, should they be changed to new extension members?
```suggestion
  - The signatures indicate if the method is a specialization (`extension(IEnumerable<int> source) { ... }`) or generic (`extension<T>(IEnumerable<T> source) { ... }`).
```